egs/wsj/s5/steps/scoring/score_kaldi_cer.sh

```diff
@@ -58,7 +58,7 @@ ref_filtering_cmd="cat"
 [ -x local/wer_output_filter ] && ref_filtering_cmd="local/wer_output_filter"
 [ -x local/wer_ref_filter ] && ref_filtering_cmd="local/wer_ref_filter"
 hyp_filtering_cmd="cat"
-[ -x local/wer_output_filter ] && hyp_filtering_cmd="local/wer_output_filter"
+[ -x local/wer_output_filter ] && hyp_filtering_cmd="local/wer_output_filter | sed 's/ //g' | sed 's/|/ /g' "
```

Comment:
@babakrkb, you shouldn't make changes to this file, it's in the shared part of the scripts. 
I think you already said you were going to fix this, though.